Ctor 457 new pack network juniper mseries netconf (#4416)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-netconf.md

---
id: network-routers-juniper-mseries-netconf
title: Juniper M-Series Netconf
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Dépendances du connecteur de supervision

Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Juniper M-Series Netconf** 

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer: ```suggestion
depuis la page **Configuration > Connecteurs > Connecteurs de supervision** :
```

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer 2: ```suggestion
depuis la page **Configuration > Connecteurs > Connecteurs de supervision** :
```

| Nom                                          | Unité |
|:---------------------------------------------|:------|
| services.detected.count                      | count |
| status                                       | N/A   |
| *services*~service.rpm.probe.loss.percentage | %     |
| service-rpm-probe-rtt-delay-average          | N/A   |
| service-rpm-probe-rtt-delay-jitter           | N/A   |
| service-rpm-probe-rtt-delay-stdev            | N/A   |
| service-rpm-probe-prtj-delay-average         | N/A   |
| service-rpm-probe-prtj-delay-jitter          | N/A   |
| service-rpm-probe-prtj-delay-stdev           | N/A   |
| service-rpm-probe-nrtj-delay-average         | N/A   |
| service-rpm-probe-nrtj-delay-jitter          | N/A   |
| service-rpm-probe-nrtj-delay-stdev           | N/A   |

</TabItem>
<TabItem value="RSVP-sessions" label="RSVP-sessions">

| Nom                                            | Unité |
|:-----------------------------------------------|:------|
| rsvp.sessions.detected.count                   | count |
| status                                         | N/A   |
| *rsvp*#rsvp.session.lsp.traffic.bytespersecond | B/s   |

</TabItem>
</Tabs>

## Prérequis

> Le service Netconf doit être activé sur le routeur Juniper M-Series, et le routeur doit être configuré pour autoriser les connexions depuis le serveur ou le poller Centreon.

Reviewer: ```suggestion
Le service Netconf doit être activé sur le routeur Juniper M-Series, et le routeur doit être configuré pour autoriser les connexions depuis le serveur ou le collecteur Centreon.
```

> Le service Netconf doit être activé sur le routeur Juniper M-Series, et le routeur doit être configuré pour autoriser les connexions depuis le serveur ou le poller Centreon.

Reviewer 2: ```suggestion
Le service Netconf doit être activé sur le routeur Juniper M-Series, et le routeur doit être configuré pour autoriser les connexions depuis le serveur ou le collecteur Centreon.
```

Un compte utilisateur avec les droits nécessaires pour accéder aux données Netconf doit également être créé.

## Installer le connecteur de supervision

### Pack

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer 2: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

1. Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer 2: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
associé à sa distribution :

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-pack-network-routers-juniper-mseries-netconf
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-pack-network-routers-juniper-mseries-netconf
```

</TabItem>
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">

```bash
apt install centreon-pack-network-routers-juniper-mseries-netconf
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-pack-network-routers-juniper-mseries-netconf
```

</TabItem>
</Tabs>

2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Juniper M-Series Netconf**

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer 2: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

| CRITICALLSPSESSIONSDETECTED | Define the LSP sessions detected conditions to match for the status to be CRITICAL                                                                                                                                                    |                        |             |
| WARNINGLSPSESSIONTRAFFIC    | Define the LSP session traffic conditions to match for the status to be WARNING                                                                                                                                                       |                        |             |
| CRITICALLSPSESSIONTRAFFIC   | Define the LSP session traffic conditions to match for the status to be CRITICAL                                                                                                                                                      |                        |             |

> | CRITICALSTATUS              | Define the conditions to match for the status to be CRITICAL (default: '%\{lspState\} !~ /up/i'). You can use the following variables: C\<%\{type\}\>, C\<%\{name\}\>, C\<%\{srcAddress\}\>, C\<%\{dstAddress\}\>, C\<%\{lspState\}\> | %\{lspState\} !~ /up/i |             |

Reviewer: C?

> | CRITICALSTATUS              | Define the conditions to match for the status to be CRITICAL (default: '%\{lspState\} !~ /up/i'). You can use the following variables: C\<%\{type\}\>, C\<%\{name\}\>, C\<%\{srcAddress\}\>, C\<%\{dstAddress\}\>, C\<%\{lspState\}\> | %\{lspState\} !~ /up/i |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS              | Define the conditions to match for the status to be CRITICAL (default: '%\{lspState\} !~ /up/i'). You can use the following variables: <%\{type\}\>, <%\{name\}\>, <%\{srcAddress\}\>, <%\{dstAddress\}\>, <%\{lspState\}\> | %\{lspState\} !~ /up/i |             |
```

> | WARNINGSTATUS               | Define the conditions to match for the status to be WARNING. You can use the following variables: <%\{type\}\>, <%\{name\}\>, <%\{srcAddress\}\>, <%\{dstAddress\}\>, <%\{lspState\}\>                                      |                        |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS              | Define the conditions to match for the status to be CRITICAL (default: '%\{lspState\} !~ /up/i'). You can use the following variables: %\{type\}, %\{name\}, %\{srcAddress\}, %\{dstAddress\}, %\{lspState\} | %\{lspState\} !~ /up/i |             |
| WARNINGSTATUS               | Define the conditions to match for the status to be WARNING. You can use the following variables: %\{type\}, %\{name\}, %\{srcAddress\}, %\{dstAddress\}, %\{lspState\}                                      |                        |             |
```

> | WARNINGSTATUS               | Define the conditions to match for the status to be WARNING. You can use the following variables: C\<%\{type\}\>, C\<%\{name\}\>, C\<%\{srcAddress\}\>, C\<%\{dstAddress\}\>, C\<%\{lspState\}\>                                      |                        |             |

Reviewer: C?

> | WARNINGSTATUS               | Define the conditions to match for the status to be WARNING. You can use the following variables: C\<%\{type\}\>, C\<%\{name\}\>, C\<%\{srcAddress\}\>, C\<%\{dstAddress\}\>, C\<%\{lspState\}\>                                      |                        |             |

Reviewer 2: ```suggestion
| WARNINGSTATUS               | Define the conditions to match for the status to be WARNING. You can use the following variables: <%\{type\}\>, <%\{name\}\>, <%\{srcAddress\}\>, <%\{dstAddress\}\>, <%\{lspState\}\>                                      |                        |             |
```

| CRITICALSERVICERPMPROBERTTDELAYSTDEV    | Define the service RPM probe route-trip time delay standard deviation conditions to match for the status to be CRITICAL                                                                                             |                   |             |
| WARNINGSERVICESDETECTED                 | Define the services detected conditions to match for the status to be WARNING                                                                                                                                       |                   |             |
| CRITICALSERVICESDETECTED                | Define the services detected conditions to match for the status to be CRITICAL                                                                                                                                      |                   |             |
| WARNINGSTATUS                           | Define the conditions to match for the status to be WARNING. You can use the following variables: <%\{testName\}\>, <%\{probeType\}\>, <%\{sourceAddress\}\>, <%\{targetAddress\}\>, <%\{probeStatus\}\>  |                   |             |

> | CRITICALSTATUS                          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: <%\{testName\}\>, <%\{probeType\}\>, <%\{sourceAddress\}\>, <%\{targetAddress\}\>, <%\{probeStatus\}\> |                   |             |

Reviewer 2: ```suggestion
| WARNINGSTATUS                           | Define the conditions to match for the status to be WARNING. You can use the following variables: %\{testName\}, %\{probeType\}, %\{sourceAddress\}, %\{targetAddress\}, %\{probeStatus\}  |                   |             |
| CRITICALSTATUS                          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %\{testName\}, %\{probeType\}, %\{sourceAddress\}, %\{targetAddress\}, %\{probeStatus\} |                   |             |
```

> | CRITICALSTATUS                          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: C\<%\{testName\}\>, C\<%\{probeType\}\>, C\<%\{sourceAddress\}\>, C\<%\{targetAddress\}\>, C\<%\{probeStatus\}\> |                   |             |

Reviewer: C?

> | CRITICALSTATUS                          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: C\<%\{testName\}\>, C\<%\{probeType\}\>, C\<%\{sourceAddress\}\>, C\<%\{targetAddress\}\>, C\<%\{probeStatus\}\> |                   |             |

Reviewer 2: ```suggestion
| WARNINGSTATUS                           | Define the conditions to match for the status to be WARNING. You can use the following variables: <%\{testName\}\>, <%\{probeType\}\>, <%\{sourceAddress\}\>, <%\{targetAddress\}\>, <%\{probeStatus\}\>  |                   |             |
| CRITICALSTATUS                          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: <%\{testName\}\>, <%\{probeType\}\>, <%\{sourceAddress\}\>, <%\{targetAddress\}\>, <%\{probeStatus\}\> |                   |             |
```

#### Découverte de services

| Nom de la règle                          | Description                                                                                       |
|:-----------------------------------------|:--------------------------------------------------------------------------------------------------|
| Net-Juniper-Mseries-Netconf-Bgp          |                                                                                                   |
| Net-Juniper-Mseries-Netconf-Disks        | Découvre les partitions du disque en utilisant son nom et supervise l'espace occupé               |
| Net-Juniper-Mseries-Netconf-Interfaces   | Découvre les interfaces réseau en utilisant leur nom et supervise leur statut et leur utilisation |
| Net-Juniper-Mseries-Netconf-Ldp          |                                                                                                   |
| Net-Juniper-Mseries-Netconf-Lsp          |                                                                                                   |
| Net-Juniper-Mseries-Netconf-Rsvp         |                                                                                                   |

> | Net-Juniper-Mseries-Netconf-Services-Rpm | Discover services and monitor their system usage                                                  |

Reviewer 2: ```suggestion
| Net-Juniper-Mseries-Netconf-Bgp               | Découvre les pairs BGP et les supervise                                                     |
| Net-Juniper-Mseries-Netconf-Disks             | Découvre les partitions du disque et supervise l'espace occupé               |
| Net-Juniper-Mseries-Netconf-Interfaces      | Découvre les interfaces réseau et supervise leur statut et leur utilisation |
| Net-Juniper-Mseries-Netconf-Ldp                | Découvre les sessions LDP et supervise leur statut                                   |
| Net-Juniper-Mseries-Netconf-Lsp                 | Découvre les LSP et supervise leur statut                                                  |
| Net-Juniper-Mseries-Netconf-Rsvp               | Découvre les sessions RSVP et les supervise                                             |
| Net-Juniper-Mseries-Netconf-Services-Rpm | Découvre les services RPM et les supervise                                              |
```

| status                                       | N/A   |
| *services*~service.rpm.probe.loss.percentage | %     |
| service-rpm-probe-rtt-delay-average          | N/A   |
| service-rpm-probe-rtt-delay-jitter           | N/A   |
| service-rpm-probe-rtt-delay-stdev            | N/A   |
| service-rpm-probe-prtj-delay-average         | N/A   |
| service-rpm-probe-prtj-delay-jitter          | N/A   |
| service-rpm-probe-prtj-delay-stdev           | N/A   |
| service-rpm-probe-nrtj-delay-average         | N/A   |
| service-rpm-probe-nrtj-delay-jitter          | N/A   |
| service-rpm-probe-nrtj-delay-stdev           | N/A   |

</TabItem>
<TabItem value="RSVP-sessions" label="RSVP-sessions">

| Nom                                            | Unité |
|:-----------------------------------------------|:------|
| rsvp.sessions.detected.count                   | count |
| status                                         | N/A   |
| *rsvp*#rsvp.session.lsp.traffic.bytespersecond | B/s   |

</TabItem>
</Tabs>

## Prérequis

*Specify prerequisites that are relevant. You may want to just provide a link\n\
to the manufacturer official documentation BUT you should try to be as complete\n\

> as possible here as it will save time to everybody.*

Reviewer 2: Prérequis à ajouter

| WARNINGBGPPEERDETECTED           | Warning threshold for number of BGP peers detected                                                                                                                                                                        |                                  |             |
| CRITICALBGPPEERDETECTED          | Critical threshold for number of BGP peers detected                                                                                                                                                                       |                                  |             |
| WARNINGBGPPEERRIBPREFIXESACTIVE  |                                                                                                                                                                                                                           |                                  |             |

> | CRITICALBGPPEERRIBPREFIXESACTIVE |                                                                                                                                                                                                                           |                                  |             |

Reviewer 2: ```suggestion
| WARNINGBGPPEERRIBPREFIXESACTIVE  | Threshold                                                                                                                                                                                                                          |                                  |             |
| CRITICALBGPPEERRIBPREFIXESACTIVE | Threshold                                                                                                                                                                                                                          |                                  |             |
```
